.gitignore

```diff
@@ -0,0 +1,71 @@
+# Melos-generated pubspec overrides
+pubspec_overrides.yaml
+
+# See https://www.dartlang.org/guides/libraries/private-files
+
+# Files and directories created by pub
+.dart_tool/
+.packages
+build/
+# If you're building an application, you may want to check-in your pubspec.lock
+pubspec.lock
+
```

Comment:
Possibly everything below this line should be removed and individual package should decide what they want to ignore. But I don't know what is the generally accepted approach in such situations

Reply:
Sure, I agree. Let's keep this gitignore as light as possible. I'll remove everything that isn't related to this monorepo root/melos itself